recipe/meta.yaml

```diff
@@ -50,7 +49,10 @@ test:
     - kedro
   commands:
     - kedro --help
-    - pip check
+# Disabled pip check because of pip-tools and rope
```

Comment:
Ah nice, I didn't know how to make the builds pass without the other packages being published to conda with the latest version!